var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -15,7 +15,7 @@ class Kokkos(CMakePackage, CudaPackage, ROCmPackage):
 
     homepage = "https://github.com/kokkos/kokkos"
     git      = "https://github.com/kokkos/kokkos.git"
-    url      = "https://github.com/kokkos/kokkos/archive/3.5.00.tar.gz"
+    url      = "https://github.com/kokkos/kokkos/archive/3.6.00.tar.gz"
```

Comment:
FYI. Changing the version here isn't necessary as Spack will automatically perform that conversion for you.